services/alb.py

```diff
@@ -2,36 +2,122 @@
+        else: non_compliant_resources.append(load_balancer['LoadBalancerArn'])
     return RuleCheckResult(
-        passed=False, compliant_resources=[], non_compliant_resources=[]
+        passed=not non_compliant_resources,
+        compliant_resources=compliant_resource,
+        non_compliant_resources=non_compliant_resources,
     )
 
 
 def elb_cross_zone_load_balancing_enabled():
+    load_balancers = client.describe_load_balancers()
+    compliant_resource = []
+    non_compliant_resources = []
+    for load_balancer in load_balancers['LoadBalancers']:
+        response = client.describe_load_balancer_attributes(
+            LoadBalancerArn=load_balancer['LoadBalancerArn']
+        )
+        result = [
+            attribute
+            for attribute in filter(
+                lambda x: x['Key'] == "load_balancing.cross_zone.enabled"
+                and x['Value'] == "true",
+                response['Attributes'],
+            )
+        ]
+        if result: compliant_resource.append(load_balancer['LoadBalancerArn'])
+        else: non_compliant_resources.append(load_balancer['LoadBalancerArn'])
     return RuleCheckResult(
-        passed=False, compliant_resources=[], non_compliant_resources=[]
+        passed=not non_compliant_resources,
+        compliant_resources=compliant_resource,
+        non_compliant_resources=non_compliant_resources,
     )
 
 
 def elb_deletion_protection_enabled():
+    load_balancers = client.describe_load_balancers()
+    compliant_resource = []
+    non_compliant_resources = []
+    for load_balancer in load_balancers['LoadBalancers']:
+        response = client.describe_load_balancer_attributes(
+            LoadBalancerArn=load_balancer['LoadBalancerArn']
+        )
+        result = [
+            attribute
+            for attribute in filter(
+                lambda x: x['Key'] == "deletion_protection.enabled"
+                and x['Value'] == "true",
+                response['Attributes'],
+            )
+        ]
+        if result: compliant_resource.append(load_balancer['LoadBalancerArn'])
+        else: non_compliant_resources.append(load_balancer['LoadBalancerArn'])
     return RuleCheckResult(
-        passed=False, compliant_resources=[], non_compliant_resources=[]
+        passed=not non_compliant_resources,
+        compliant_resources=compliant_resource,
+        non_compliant_resources=non_compliant_resources,
     )
 
 
 def elb_logging_enabled():
+    load_balancers = client.describe_load_balancers()
+    compliant_resource = []
+    non_compliant_resources = []
+    for load_balancer in load_balancers['LoadBalancers']:
+        response = client.describe_load_balancer_attributes(
+            LoadBalancerArn=load_balancer['LoadBalancerArn']
+        )
+        result = [
+            attribute
+            for attribute in filter(
+                lambda x: x['Key'] == "connection_logs.s3.enabled"
```

Comment:
```suggestion
                lambda x: x['Key'] == "access_logs.s3.enabled"
```

`connection_logs.s3.enabled`가 아닌 `access_logs.s3.enabled`를 확인해야 합니다.
https://docs.aws.amazon.com/config/latest/developerguide/elb-logging-enabled.html